### ISSUE TITLE
[dashboard: recent applications]: empty spot where calamares installer was before

### ISSUE THREAD

**Issue by star-buck** [blue-systems/plasma-5.5#11]
lets remove any empty spots for removed apps?
![installer-empty-slot](https://cloud.githubusercontent.com/assets/840070/12076712/ba6d62ee-b16b-11e5-8b6e-7d075cf7ac4d.png)

**Comment by eikehein** [comment]
Already done recently :) (listed in weekly work logs in meeting notes).

**Comment by star-buck** [comment]
Which version? Also, lets close tickets here as always only after confirmed testing of a shipped installation.

**Comment by eikehein** [comment]
5.5.2

**Comment by star-buck** [comment]
fixed
